gammapy/scripts/cta_irf.py

```diff
@@ -0,0 +1,105 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+import astropy.units as u
+from astropy.table import Table
+from ..utils.fits import get_hdu
+from ..irf import EffectiveAreaTable2D
+from ..background import Cube
+from ..irf import EnergyDispersion2D
+from ..irf import EnergyDependentMultiGaussPSF
+
+__all__ = [
+    'CTAIrf',
+]
+
+
+class CTAIrf(object):
+    """Contain CTA instrument response functions.
+
+    Class handling CTA instrument response function. For now
+    we use the production 2 of the CTA IRF
+    (https://portal.cta-observatory.org/Pages/CTA-Performance.aspx)
+    adapted from the ctools
+    (http://cta.irap.omp.eu/ctools/user_manual/getting_started/response.html).
+    The IRF format should be complient compliant with the one discussed
+    at http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/.
+    Waiting for a new public production of the CTA IRF,
+    we'll fix the missing pieces.
+
+    Parameters
+    ----------
+    area_2d : `~gammapy.irf.EffectiveAreaTable2D`
+        2D Effective Area Table
+    bcg_cube : `~gammapy.background.Cube`
+        Cube container class with scheme fixed to `bg_cube`
+    edisp_2d : `~gammapy.irf.EnergyDispersion2D`
+        Offset-dependent energy dispersion matrix
+    psf_mgauss : `~gammapy.irf.EnergyDispersion2D`
+        Position-dependent multi-Gauss PSF
+    """
+
+    def __init__(self, area_2d=None, bkg_cube=None, e_disp=None, psf_mgauss=None):
+        self.area_2d = area_2d
+        self.bkg_cube = bkg_cube
+        self.e_disp_2d = e_disp_2d
+        self.psf_mgauss = psf_mgauss
+
+    @classmethod
+    def from_fits(cls, irf_path):
```

Comment:
Rename `from_fits` -> `read`.
We always use `read` for the method that takes a filename.
`from_fits` is used for constructor methods that take `astropy.io.fits` objects if present.

Parameter is called `irf_path` and then `path` in the docstring. Usually we put `filename`.